samples/bluetooth/ipsp/README.rst

```diff
@@ -27,14 +27,132 @@ following commands:
+be run as root, and you will need to break it manually):
+
+.. code-block:: console
+
+   # hcitool lescan
+   LE Scan ...
+   CB:AF:14:57:D8:6E (unknown)
+   CB:AF:14:57:D8:6E Test IPSP node
+
+After you found out BLE address, connect to the board:
 
 .. code-block:: console
 
    $ echo "connect <bdaddr> <type>" > /sys/kernel/debug/bluetooth/6lowpan_control
 
-Once connected a dedicated interface will be created, usually bt0, which can
-then be used as following:
+Where ``<bdaddr>`` is BLE address and ``<type>`` is BLE address type, 1 for public
+address and 2 for static random address. As you can see from the IPSP sample
+output above, it uses the latter. So, with the sample output above, the command
+will be:
+
+.. code-block:: console
+
+   $ echo "connect CB:AF:14:57:D8:6E 2" > /sys/kernel/debug/bluetooth/6lowpan_control
+
+Once connected a dedicated interface will be created, usually bt0. You can verify this
+with the following command:
+
+.. code-block:: console
+
+   # ifconfig
+   bt0       Link encap:UNSPEC  HWaddr F8-2F-A8-FF-FE-EB-6D-8C-00-00-00-00-00-00-00-00
+             inet6 addr: fe80::fa2f:a8ff:feeb:6d8c/64 Scope:Link
+             UP POINTOPOINT RUNNING MULTICAST  MTU:1280  Metric:1
+             RX packets:2 errors:0 dropped:3 overruns:0 frame:0
+             TX packets:6 errors:0 dropped:0 overruns:0 carrier:0
+             collisions:0 txqueuelen:1000
+             RX bytes:92 (92.0 B)  TX bytes:233 (233.0 B)
+
+As can be seen from the output, only a link-local IPv6 address was assigned
+to the interface.
+
+At this point, you can already test IPv6 connectivity (and discover your board's IPv6
+address) by pinging "All local-link nodes" IPv6 address:
+
+.. code-block:: console
+
+   # ping6 -I bt0 ff02::1
+   PING ff02::1(ff02::1) from fe80::fa54:a8ff:feeb:218f bt0: 56 data bytes
+   64 bytes from fe80::fa54:a8ff:feeb:218f: icmp_seq=1 ttl=64 time=0.088 ms
+   64 bytes from fe80::c9af:14ff:fe57:d86e: icmp_seq=1 ttl=64 time=285 ms (DUP!)
+
+For each ping packet, both your host and BLE board send reply, so you can see
+board's reply marked as "DUP!". We can now ping the board directly:
+
+.. code-block:: console
+
+   # ping6 fe80::c9af:14ff:fe57:d86e%bt0
+   PING fe80::c9af:14ff:fe57:d86e%bt0(fe80::c9af:14ff:fe57:d86e) 56 data bytes
+   64 bytes from fe80::c9af:14ff:fe57:d86e: icmp_seq=1 ttl=64 time=177 ms
+   64 bytes from fe80::c9af:14ff:fe57:d86e: icmp_seq=2 ttl=64 time=53.0 ms
+
+Note that command above uses "scoped IPv6 address", where scope is defined by
+networking interface, with ``%bt0`` appended in this case. Specification like
+that is in alternative to passing ``-I bt0`` to ``ping6`` (and works with
+other networking tools like ``telnet``, ``nc``, ``curl``, etc.)
+
+While we can use link-local address, it's not very convenient, as it must be
+scoped and changes on each run. Instead, IPSP sample is configured with
+``2001:db8::1`` static address, so let's configure host's interface to let
```

Comment:
static address and we'll configure the host's interface to access that address by
configuring bt0 with address...